### ISSUE TITLE
Timeout Error

### ISSUE THREAD

**Issue by markito12** [BeepIsla/csgo-commend-bot#577]
**Detailed question description**
Tried running commend bot and only get timeout errors.

**Config**
```
{
	"commend": {
			"friendly": 9,
			"teaching":9,
			"leader":9
	},
	"report": {
			"aimbot": 10,
			"wallhack": 10,
			"speedhack": 10,
			"teamharm": 10,
			"abusive": 10
	},
	"account": {
			"username": "user",
			"password": "pass",
			"sharedSecret": ""
	},
	"fetcher": {
			"username": "user",
			"password": "pass",
			"sharedSecret": "",
			"maxTries": 10,
			"tryDelay": 2
500,
			"askSteamGuard": true
	},
	"proxy": {
			"enabled": false,
			"file": "proxies.txt",
			"switchProxyEveryXaccounts": 20
	},
	"type": "COMMEND",
	"method": "SERVER",
	"target": "https://steamcommunity.com/id/ID",
	"serverID": "WorkingServer",
	"matchID": "0",
	"perChunk": 20,
	"betweenChunks": 300000,
	"cooldown": 28800000,
	"showCommends": true,
	"autoReportOnMatchEnd": false,
	"switchServerAfterChunks": 5,
	"protocol": "Auto",
	"steamWebAPIKey": "38**************************",
	"disableUpdateCheck": false
}
```

**Version**
- NodeJS: `v14.0.0`
- Script version: `2.10.0`

![Screenshot_20](https://user-images.githubusercontent.com/31602075/89598697-d4012d00-d833-11ea-90f9-c3eb038afb05.png)

**Comment by markito12** [comment]
> **Detailed question description**
> Tried running commend bot and only get timeout errors.
> 
> **Config**
> 
> ```
> {
> 	"commend": {
> 			"friendly": 9,
> 			"teaching":9,
> 			"leader":9
> 	},
> 	"report": {
> 			"aimbot": 10,
> 			"wallhack": 10,
> 			"speedhack": 10,
> 			"teamharm": 10,
> 			"abusive": 10
> 	},
> 	"account": {
> 			"username": "user",
> 			"password": "pass",
> 			"sharedSecret": ""
> 	},
> 	"fetcher": {
> 			"username": "user",
> 			"password": "pass",
> 			"sharedSecret": "",
> 			"maxTries": 10,
> 			"tryDelay": 2
> 500,
> 			"askSteamGuard": true
> 	},
> 	"proxy": {
> 			"enabled": false,
> 			"file": "proxies.txt",
> 			"switchProxyEveryXaccounts": 20
> 	},
> 	"type": "COMMEND",
> 	"method": "SERVER",
> 	"target": "https://steamcommunity.com/id/ID",
> 	"serverID": "WorkingServer",
> 	"matchID": "0",
> 	"perChunk": 20,
> 	"betweenChunks": 300000,
> 	"cooldown": 28800000,
> 	"showCommends": true,
> 	"autoReportOnMatchEnd": false,
> 	"switchServerAfterChunks": 5,
> 	"protocol": "Auto",
> 	"steamWebAPIKey": "38**************************",
> 	"disableUpdateCheck": false
> }
> ```
> 
> **Version**
> 
> * NodeJS: `v14.0.0`
> * Script version: `2.10.0`
> 
> ![Screenshot_20](https://user-images.githubusercontent.com/31602075/89598697-d4012d00-d833-11ea-90f9-c3eb038afb05.png)

* serverID: (Community Server format ID:PORT)

**Comment by markito12** [comment]
fixed using version 2.6.1
